test/shared/src/main/scala/zio/test/Gen.scala

```diff
@@ -478,7 +501,7 @@ object Gen extends GenZIO with FunctionVariants with TimeVariants {
    */
   def fromIterable[R, A](
     as: Iterable[A],
-    shrinker: (A => ZStream[R, Nothing, A]) = defaultShrinker
+    shrinker: A => ZStream[R, Nothing, A] = defaultShrinker
```

Comment:
Stray change?

Reply:
Minor refactoring.
Shouldn't I commit this change in this PR?